dracut/30ignition/module-setup.sh

```diff
@@ -112,5 +114,8 @@ installkernel() {
      instmods -c vsock
      instmods -c vmw_vsock_virtio_transport_common
      instmods -c vmw_vsock_virtio_transport
+
+     # required for cex card early initialization.
+     instmods -c zcrypt_cex4
```

Comment:
This is a kernel module. On Fedora at least, it's part of `kernel-modules-core`. So this isn't strictly blocked by https://github.com/coreos/fedora-coreos-tracker/issues/1708.

CI is failing because that kernel module is not available on x86_64. We need to make this `instmods` conditional on s390x. See e.g. https://github.com/dracutdevs/dracut/blob/4980bad34775da715a2639b736cba5e65a8a2604/modules.d/90kernel-modules/module-setup.sh#L63 as an example from dracut itself.

Reply:
Thank you...